data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+
+
+
+mission "Unfettered: Bring Detainer"
+	landing
+	name `Recover a detainer`
+	description `He Yi has asked you to earn a translation device by bringing back a powerful alien weapon, said to be able to neutralize the engines of its target. You should probably look around Kor Sestor space.`
+	source "Warfeed"
+	to offer
+		has "Unfettered: Know Your Enemy: active"
+		has "First Contact: Wanderer: offered"
+		not "language: Wanderer"
+	on offer
+		conversation
+			`You go find He Yi in his war room, who does not seem too surprised to see you. "I expected you to come here. You must want one of us to act as 'diplomat' with the aliens to the north."`
+			`	You nod.`
+			`	"Well, that's not our way. We do not ask. We take. Things need to be earned, in battle or otherwise, or they have no value. I have obtained a device that is able to translate your backwards language into ours. Because these birds speak Hai, you will be able to communicate with them later.`
+			`	"If you want this translator, you will have to earn it. There is something I want: I have heard tales of a weapon that would neutralize one's engines. Such a weapon would be a very nice addition to both my collection and arsenal during battle. Bring me one of these weapons, and you will have earned this translator."`
+			choice
+				`	"Sure. I'll go and bring it back for you."`
+					accept
+				`	"Where can I find said weapon?"`
+			`	"The green humans told me of it, and we know that they were active in the territory to the east. You would do well to go and look there. There are legends speaking of an everlasting battlefield - one that never ends the carnage, never sleeps, as if gods were laughing in the background, enjoying the endless show. The Alphas did not tell me much though... I heard it in a bar, so it may be a bit exaggerated," he finishes with a smirk.`
+			choice
+				`	"Sure, I will go and bring it back for you."`
+					accept
+				`	"I doubt I have the strength to face the Korath machines."`
+			`	"If you cannot acquire the weapon, then you do not deserve this device. There should be other ways to get it than brute force, however. You could loot a ship while its allies are dustracted. Use whatever tools you have at your disposal. What do you say?"`
+			choice
+				`	"Sure, I will go and bring it back for you."`
+					accept
+				`	"This is way too risky. I'm out."`
+			action
+				fail "Unfettered: Know Your Enemy"
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on visit
+		dialog `You do not yet have the weapon He Yi asked for. Investigate the battles of the Korath automata and you may see it in action.`
+	on complete
+		outfit "Korath Detainer" -1
+		set "language: Wanderer"
+		dialog `He Yi is impressed that you managed to bring this detainer to him, and happily exchanges it for a translator. He says you should still go to his herald to bring him news of this, and the scans for a reward.`
```

Comment:
Will this `on complete` appear in the same planet visit/instance as the `on complete` on line 3540 (the previous mission)? It seems to me like you might have to take off and land for it to trigger. If it does happen in the same planet visit, will this `dialog` trigger before the other `dialog`?

Reply:
One is on Firelode, the other on Warfeed